tlock.go

```diff
@@ -166,9 +190,16 @@ func CiphertextToBytes(ciphertext *ibe.Ciphertext) ([]byte, error) {
 
 // BytesToCiphertext converts bytes to a ciphertext.
 func BytesToCiphertext(b []byte) (*ibe.Ciphertext, error) {
-	expLen := kyberPointLen + cipherVLen + cipherWLen
-	if len(b) != expLen {
-		return nil, fmt.Errorf("incorrect length: exp: %d got: %d", expLen, len(b))
+	expLenOnG1 := kyberG1PointLen + cipherVLen + cipherWLen
```

Comment:
Kyber provides an `.ScalarLen()` - is that different?

Reply:
The length of V and W in the ciphertext are defined by the length of the message being fed at the IBE Kyber level, we can have the constant live in tlock, it's not the same as ScalarLen().